functions/api/branch.js

```diff
@@ -41,158 +40,47 @@ router.get("/", async (request, response) => {
  * @returns Json object containing requested branch
  * @throws 400 if the branch does not exists in firestore
  */
-router.get('/:branch', async (request, response, next) => {
-    var requestedBranch = request.params.branch.toLocaleLowerCase()
-    logger.info(`Retrieving branch ${requestedBranch} from firestore`)
-    const doc = db.collection(constants.BRANCHES).doc(requestedBranch);
+router.get('/:id', async (request, response, next) => {
+    var branchId = request.params.id
+    logger.info(`Retrieving branch from firestore`)
+    const doc = db.collection(constants.BRANCHES).doc(branchId);
     const branch = await doc.get()
     if (!branch.exists) {
-        const error = new Error(`Requested branch ${requestedBranch} is not present in Firestore.`)
+        const error = new Error(`Requested branch is not present in Firestore.`)
         error.statusCode = 404
         next(error)
         return;
     }
     var branchData = branch.data()
-    branchData[constants.BRANCH] = utils.capitalize(branch.id)
+    branchData[constants.ID] = branchId
+    branchData[constants.NAME] = utils.capitalize(branchData[constants.NAME])
```

Comment:
do we have to capitalize?

Reply:
Vinayak wants the label to show. I capitalize to make labels uniform

Reply:
i havent :) Lets see